internal/pkg/aws/ec2/ec2.go

```diff
@@ -83,8 +84,34 @@ func New(s *session.Session) *EC2 {
 	}
 }
 
-// ListVPC returns IDs of all VPCs.
-func (c *EC2) ListVPC() ([]string, error) {
+// VPC contains the ID and name of a VPC.
+type VPC struct {
+	ID   string
+	Name string
+}
+
+// String formats the elements of a VPC into a display-ready string.
+func (v *VPC) String() (string, error) {
+	if v.Name != "" {
+		return fmt.Sprintf("%s (%s)", v.ID, v.Name), nil
+	}
+	return v.ID, nil
+}
+
+// ExtractVPC extracts the VPC ID from the VPC display string.
+func ExtractVPC(label string) (VPC, error) {
```

Comment:
I think we need this error but it always returns `nil` for now. In case we won't panic we should probably check the length of array before we refer them as `[0]` or `[1]`

Reply:
Made error handling more robust (ie actually catch something) but tabled the regex in a TODO.